dapr/proto/runtime/v1/dapr.proto

```diff
@@ -711,6 +714,16 @@ message TerminateWorkflowRequest {
 }
 
 message TerminateWorkflowResponse {
+  // Empty
+}
+
+message PurgeWorkflowRequest {
+  string instance_id = 1;
+  string workflow_component = 2;
```

Comment:
`json_name` tags are very important when the proto contains `_`. Otherwise, the default when serializing as JSON (like in Universal APIs) is to use the key in snake_case.

```suggestion
  string instance_id = 1 [json_name = "instanceId"];
  string workflow_component = 2 [json_name = "workflowComponent"];
```

Reply:
nit: use `instanceID` instead of `instanceId` for the JSON name? This is what I've been trying to do in my other PRs.